Fits/Dreadnoughts/[FL33T] Anti-Cap Naglfar V2.md

```diff
@@ -8,41 +8,41 @@ Used to provide high DPS against other capital ships.  Highly ineffective for us
 
 ## Fit
 ```
-[Naglfar, T2 Anti-Cap Naglfar]
+[Naglfar, Anti-Cap Naglfar V2]
```

Comment:
Do we get much out of a tracking enhancer over PDU + T2 extenders?

Reply:
Primary idea with the TE is dropping it at range with the Phoenix and having them use Barrage instead of Hail. The DPS is still very good at 45kish and it's going to keep them on grid for waaaay longer doing more overall damage than dropping them close and having them eat shit. It just goes back to the primary weakness of shield dreads being capacitor more than anything else, so getting these out of neut range of enemy dreads is super important.